api/server.js

```diff
@@ -0,0 +1,18 @@
+const cors = require('cors');
+const express = require('express');
+const app = express()
+const port = 8000
```

Comment:
port standard is 3000 not 8000
to make sure that server works when deployed better use : 

const port = process.env.PORT || 3000;